src/main/scala/nl/knaw/dans/easy/stage/Settings.scala

```diff
@@ -47,8 +47,6 @@ object Settings {
              urn: String,
              doi: String,
              otherAccessDoi: Boolean,
-             stageFileDataAsRedirectDatastreams: Boolean,
-             fileDataRedirectBaseUrl: Option[URL],
```

Comment:
link gives a 404, probably because your fork of easy-ingest-flow is private (as is the whole project 😞